### ISSUE TITLE
Permission Problems / Thrustmaster T248 / Pop!_OS

### ISSUE THREAD

**Issue by MicHaeL-MonStaR** [berarma/oversteer#173]
I have an issue with permissions, with Oversteer saying that "I don't have the required permissions to change my wheel settings.". - While it does work for most settings (some are unavailable), it's also inconsistent, in that one profile might have access to more settings than another. (For example, profiles I made before, even days ago, allowed me to change the rotation-range, but a new profile I just made does not.)

So clearly I need to do something about the permission-rules. I just don't completely understand the instructions on the main page.

It says that udev-rules will be installed automatically, and indeed there are files for wheels in `/usr/local/lib/udev/rules.d`, of which the Thrustmaster one contains my model, the T248.

However, below that it says that I should check for older rules in `/etc/udev/rules.d` or `/lib/udev/rules.d` if there are any issues with permissions. - But what rules, though? Just any in general or specific to Oversteer or steering-wheels? - Because in the former folder there's only a file for _openrgb_, and in the latter there's many files, none of which carry the name "oversteer" or "thrustmaster".

What can/should I do?

**Comment by MicHaeL-MonStaR** [comment]
Sorry, it seems I've found the issue through https://github.com/berarma/oversteer/issues/106#issuecomment-1163441366, which did indeed work.
I just had to figure out how to paste into the target folder/directory, as I didn't have permission, but I had to open the browser as root, which I didn't know how to do at first.

Oversteer now doesn't tell me I don't have permission anymore, but I also don't know how it changed in the first place (as it didn't have the issue before). - But it works.
